pages/api/movies.js

```diff
@@ -0,0 +1,28 @@
+import dbConnect from '../../lib/dbConnect';
+import Movie from '../../models/movie';
+
+export default async function handler(req, res) {
+  await dbConnect();
+
+  if (req.method === 'GET') {
+    try {
+      const movies = await Movie.find(
+        {}
+      ); /* find all the data in our database */
+      res.status(200).json({ success: true, data: movies });
+    } catch (error) {
+      res.status(400).json({ success: false });
```

Comment:
Nothing that really needs a review, just a thought that popped up: is 400 / bad request appropriate?